main.go

```diff
@@ -49,13 +49,11 @@ func main() {
 	}
 	for _, config := range configs {
 		log.Debug("Bug configs", "Appending processSpaces() with configID: %v and refresh interval %v", config.ConfigId, config.RefreshInterval)
-		if config.Enable != nil && *config.Enable {
-			functions = append(functions, common.LoopWithParam(processSpaces, config.ConfigId, time.Duration(config.RefreshInterval)*time.Second))
-		}	
+			functions = append(functions, common.LoopWithParam(processSpaces, config.ConfigId, time.Duration(config.RefreshInterval)*time.Second))	
 	}
 	common.WaitFor(functions...)
 	for _, config:= range configs {
-		config.Active = false
+		conf.SetConfigActiveState(context.Background(), config, false)
```

Comment:
config.Active = false is doublicated to set the state to true with method parameter. It will be better to change the method parameter from complete "config" to configId only. The configId is the only value you use in this method.